.travis.yml

```diff
@@ -27,3 +27,4 @@ script:
     - export CGO_LDFLAGS="${CGO_LDFLAGS} -L${TRAVIS_BUILD_DIR}/build_deps/TileDB/dist/lib"
     - scripts/travis.gofmt.sh
     - go test -v ./...
+    - go test -v ./examples/...
```

Comment:
You don't need include `./examples/...`, the `./...` will run all packages and subpackages excluding the vendor folder.

Reply:
fixed